cf-java-logging-support-core/src/main/java/com/sap/hcp/cf/logging/common/VcapEnvReader.java

```diff
@@ -85,7 +90,7 @@ public static void getAppInfos(Map<String, String> tags, Set<String> envKeys) {
                 addField(tags, envKeys, envMap, CF_ORGANIZATION_ID, Fields.ORGANIZATION_ID);
                 addField(tags, envKeys, envMap, CF_ORGANIZATION_NAME, Fields.ORGANIZATION_NAME);
             } catch (Exception ex) {
-                LoggerFactory.getLogger(VcapEnvReader.class).error("Cannot get infos from environment", ex);
+                LoggerHolder.LOG.error("Cannot get infos from environment", ex);
```

Comment:
I think here the original intention is less prominent. The logger would only be created, if an exception occurred. In that case a stacktrace is generated anyhow. Still a static instance may be less resource intensive.

Reply:
In the log4j case, two stack traces are created.
 
In other locations the comment 
`/* -- avoids substitute logger warnings on startup -- */`
was made - therefore I used the same pattern.

However, i'll change this in a further commit to the simple static solution.

Reply:
Thanks for pointing that out. I was unaware of that problem. Please leave the holder pattern in all converters, i.e. package ` com.sap.hcp.cf.logging.common.converter`.

Reply:
The two classes not in this package. For LogContext I'm running into issues. See https://github.com/SAP/cf-java-logging-support/pull/83#issuecomment-734166802.

_Proposal_:
* LogContext -> Holder
* VcapEnvReader -> static

OK?

Reply:
That would be fine.

Reply:
done